### DISCUSSION TITLE
[Question]: Retain events only of a specific object?

### DISCUSSION THREAD

**Discussion by Charlielx** [blakeblackshear/frigate#11551] [Ask A Question]
Is there a way to entirely disable events being created for a specific type of object? I've found that you can prevent events from being stored, but the events are still being created so they show up in the log. Is there a way to disable them entirely? I'm currently using:
```
objects:
     person: 0
```

**Comment by NickM-27** [comment]
Set objects track at the camera level for the objects you want to have events for on that camera
